### DISCUSSION TITLE
Please bring back the Manga Mode in the menu

### DISCUSSION THREAD

**Discussion by tasuken** [sumatrapdfreader/sumatrapdf#2843] [Ideas]
You may think that manga mode is only for comics, but the Japanese language was originally written vertically, and even today, many Japanese books other than manga are right-bound and vertically written.

Therefore, as before, there should be an option in the View of Menu to always show the Manga mode toggle.

![image](https://user-images.githubusercontent.com/54142732/182219864-2259b4cb-4e1d-4cad-8d73-208ff9496a3c.png)
![image](https://user-images.githubusercontent.com/54142732/182220249-282a1da3-6ea7-47cd-94e6-2b4c6bb10cc7.png)

**Comment by GitHubRulesOK** [comment]
in newer versions you can add as a key choice see https://github.com/sumatrapdfreader/sumatrapdf/discussions/2775 issue is covered by https://github.com/sumatrapdfreader/sumatrapdf/issues/2258

**Comment by kjk** [comment]
Or use `Ctrl + K` and `Toggle Manga Mode` command (you can type "mang")
